### PR TITLE
HotFix - Core data crash

### PR DESCRIPTION
**Pull request by ketanshikharebsft**
Core data crash fix
In-app time interval NSNumber comparison issue fix

### CODE REVIEW

File: BlueShift-iOS-SDK/BlueShift.m [hunk 1]
```diff
@@ -151,7 +151,7 @@ - (void) setupWithConfiguration:(BlueShiftConfig *)config {
         }
         
         if ([[BlueShiftAppData currentAppData] getCurrentInAppNotificationStatus] == YES && config.inAppManualTriggerEnabled == NO) {
-            _inAppNotificationMananger.inAppNotificationTimeInterval = [NSNumber numberWithDouble:config.BlueshiftInAppNotificationTimeInterval];
+            _inAppNotificationMananger.inAppNotificationTimeInterval = config.BlueshiftInAppNotificationTimeInterval;
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/BlueShift.m:154]: there is no need to check the `.doubleValue` here, right?

File: BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationManager.h [hunk 2]
```diff
@@ -16,7 +16,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @property (nonatomic, strong, readwrite) BlueShiftNotificationViewController * _Nullable currentNotificationController;
 @property (nonatomic, weak) id<BlueShiftInAppNotificationDelegate> inAppNotificationDelegate;
-@property (nonatomic, strong) NSNumber* inAppNotificationTimeInterval;
+@property double inAppNotificationTimeInterval;
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationManager.h:19]: okay, now I understood why we don't need `.doubleValue` :)

File: BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m [hunk 3]
```diff
@@ -399,7 +399,7 @@ - (void)applyIconToLabelView:(UILabel *)iconLabelView andFontIconSize:(NSNumber
             [self createFontFile: iconLabelView];
         }
     
-        CGFloat iconFontSize = (fontSize !=nil && fontSize > [NSNumber numberWithInt:0])? fontSize.floatValue : 22.0;
+        CGFloat iconFontSize = (fontSize && fontSize.doubleValue > 0)? fontSize.floatValue : 22.0;
```
Comment (rahulrvp) [inline BlueShift-iOS-SDK/InApps/BlueShiftNotificationViewController.m:402]: why do we need to use `.doubleValue` for comparison and `.floadValue` for return value? Can we use the same code in both places?

**Review by rahulrvp — APPROVED** [review]
LGTM

File: BlueShift-iOS-SDK/BlueShift.m (hunk 1 above)
Comment (ketanshikharebsft) [reply BlueShift-iOS-SDK/BlueShift.m:154]: yes

**Pull request merged**